x/perpetual/spec/09_funding_payment.md

```diff
@@ -53,26 +53,31 @@ The distribution of funding fees is based on the calculated funding rate and ser
 
 - **Funding Rate Direction**:  
   The funding rate determines **which side of the market receives the collected funding payments**.
-  - **Positive funding rate**: Short traders receive payments.
-  - **Negative funding rate**: Long traders receive payments.
+  - **Long custody is greater than short liabilities**: Short traders receive payments.
+  - **Short liability is greater than long custody**: Long traders receive payments.
 
-However, the **actual distribution amount** is not directly proportional to the funding rate alone. Instead, the amount that each position receives is calculated based on the size of that position relative to the total liabilities on the side that is receiving the funding payments.
+However, the **actual distribution amount** is not directly proportional to the funding rate alone. Instead, the amount that each position receives is calculated based on the size of that position relative to the total liabilities/custody on the side that is receiving the funding payments.
 
 The formula for determining the amount distributed to each position is as follows:
 
 ```
 payment_to_position = (position_liabilities / pool_liabilities) * total_funding_collected
 ```
+```
+payment_to_position = (position_custody / pool_custody) * total_funding_collected
+```
```

Comment:
@amityadav0 could you clarify here that one formula is for long and the other is for short positions thanks